### PR TITLE
add custom stylesheets

### PR DESCRIPTION
**Pull request by cmkohnen**
fixes https://github.com/SjoerdB93/Graphs/issues/35
My suggestions on how the stylesheets should look.
They are currently installed in /app/lib/python3.10/site-packages/matplotlib/mpl-data/stylelib as this is where all matplotlib styles are stored and checked for. As we are only targeting flatpak this shouldn't affect any other application, and even if so, it shouldn't disrupt anything.
I also updated default config settings.
Screenshots:
grafik
grafik
Feedback appreciated

### CODE REVIEW

**Comment by SjoerdB93** [comment]
This looks nice, and especially the dark mode looks so much nicer. The only thing I'd like to add is that these styles fix the used font size for some reason. It still respects my choice of font, but the font size seems fixed. If that can be fixed this would be good to go, as the style sheet themselves are quite nice. I think the original light mode was fine as well, but this one has some added clarity and I think is good as well. But again, dark mode is a huge improvement. Also it's a big improvement to have grids on both of them (or on neither of them, at least have it consistent).

File: data/styles/adwaita.mplstyle [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# adwaita
+patch.facecolor: FAFAFA
+patch.linewidth: .3
+
+axes.axisbelow: True
+axes.edgecolor: black
+axes.facecolor: white
+axes.grid: True
+axes.labelcolor: black
+axes.labelsize: 11
```
Comment (SjoerdB93) [inline data/styles/adwaita.mplstyle:10]: I think this maybe the line that overrides the chosen font size. If we want to use 11 as default font size, we can set that in the config file.

File: data/styles/adwaita.mplstyle [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+axes.linewidth: .8
+axes.prop_cycle: cycler('color', ['4C72B0', '55A868', 'C44E52', '8172B2', 'CCB974', '64B5CD'])
+axes.titlesize: 14
+
+font.family: sans-serif
+font.sans-serif: Cantarell
+font.weight: normal
+font.size: 12
```
Comment (SjoerdB93) [inline data/styles/adwaita.mplstyle:18]: Or these... 
Basically the fonts family, sans-serif, weight and size properties can all be set in the config file. For some reason the stylesheet seem to override at least the font sizes that are chosen. Making it impossible to change the font size.

**Review by SjoerdB93 — CHANGES_REQUESTED** [review]

**Comment by cmkohnen** [comment]
The problem was, that the axes.fontsize option was never set from config, it is fixed now.

**Review by SjoerdB93 — APPROVED** [review]

**Pull request merged**